aspnetcore/toc.yml

```
@@ -744,19 +744,19 @@ items:
         items:
           - name: Overview
             uid: grpc/basics
-          - name: Create gRPC services
+          - name: Create services
             uid: grpc/services
           - name: Create Protobuf messages
             uid: grpc/protobuf
-          - name: Versioning gRPC services
+          - name: Versioning services
```

Can this be on the outside change that it helps loc ...

```suggestion
          - name: Version services
```